cleanup-workflows.sh

```
@@ -0,0 +1,47 @@
+  "core-security.yml"
+  "README.md"
+)
+
+echo "Archiving unnecessary GitHub Actions workflows..."
+echo ""
+
+# Move each workflow file to archive directory if not in preserve list
+cd "$WORKFLOWS_DIR"
+for file in *.yml; do
+  # Skip files in the preserve list
+  if [[ " ${PRESERVE_FILES[@]} " =~ " ${file} " ]]; then
```

Shellcheck (0.10.0)

[error] Arrays implicitly concatenate in [[ ]]. Use a loop (or explicit * instead of @).

(SC2199)